sdk/rust-examples/fd_create/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "fd_create"
+version = "0.1.0"
+authors = ["Edmund Grimley Evans <edmund.grimley-evans@arm.com>"]
```

Comment:
Can you attribute to the "Veracruz development team" and set the version as "0.3.0".  Also, can you provide a "description" field?

Reply:
Done.